examples/hello/go/README.md

```diff
@@ -9,10 +9,8 @@ fn init <YOUR_DOCKERHUB_USERNAME>/hello
 fn build
 # test it
 cat hello.payload.json | fn run
-# push it to Docker Hub
-fn push
 # Create a route to this function on IronFunctions
-fn routes create myapp /hello
+fn deploy myapp
```

Comment:
I thought we were discussing that deploy would be more of a bulk operation only, for when you setup an entire app structure?  What path would be used here?